config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
    root "users#index"
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+  resources :users, only: %i[index show] do
+    resources :posts, only: %i[index show new create]
+  end
+
+  resources :posts do
+    resources :comments, only: %i[create]
```

Comment:
- [ ] I would recommend adding `new`  in the comments resource. When we type `rails routes` it should be like below example code.

```ruby
 GET   /posts/:post_id/comments/new(.:format)     comments#new
 ```

Reply:
It is done, thank you @Dipeshtwis  for the review!